### ISSUE TITLE
Training loss turns to NaN

### ISSUE THREAD

**Issue by zengbohan0217** [wanglids/ST-4DGS#3]
Dear authors,
Thank you for releasing the excellent work ST-4DGS as open source. When we tried to use ST-4DGS to reconstruct our new dataset, we encountered some issues. The visualization result from the COLMAP initialization looks normal, but we are unsure whether our index.npy is correctly written. During the initial phase of training, everything proceeds without any issues, but as training continues, the loss eventually turns to NaN. We have included one of the results from the training process below. Could you kindly advise on some details regarding the COLMAP initialization?
![微信图片_20241217164909](https://github.com/user-attachments/assets/b01a273b-d21d-4676-9c00-836cb0777a34)

**Comment by wanglids** [comment]
We are very sorry about the issue with NaN. We have added abnormal detection to attempt to resolve NaN in some data during training, which may be a useful trick to solve this problem. You can download it from https://github.com/wanglids/ST-4DGS/releases
